### DISCUSSION TITLE
Enable notifications for particular workflow run

### DISCUSSION THREAD

**Discussion by melissachang** [community/community#10106] [Actions]
Right now, we can enable notifications for all workflow runs, or only failed workflows:

![image](https://user-images.githubusercontent.com/10929390/149431136-1577c5e6-b675-4df9-b211-8bbae100912f.png)

I don't want all passing emails (so I check `Send notifications for failed workflows only`).

But I want passing email for *specific workflow runs*. For example: I have a PR which has approvals. I'm just waiting for workflow to finish before merging. I'd like to get an email when workflow finishes, so I know I can merge.

Related:
https://github.com/github/feedback/discussions/9243
https://github.com/github/feedback/discussions/4369

**Comment by ethomson** [comment]
Thanks for the feedback @melissachang - I've added this to our backlog.
